### PR TITLE
Rename Images Alternative Text

### PR DESCRIPTION
**Pull request by ahmedadel56**
We reviewed all the projects of our team members and we changed the selected items of the list.

- [x] Page titles - no issues found
- [ ] Image text alternatives** (https://github.com/sergiogval/sergiogval.github.io/pull/16/commits/ddb7e5f1c5ba81ab57321d640bf6b83f1306e97c#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L20)
- [ ] Text headings (https://github.com/sergiogval/sergiogval.github.io/pull/16/commits/e622f1486c379086c844fcc62fdef32a223bc13d#)
- [ ] Color contrast - Since we are following the template color pallette, we take this as if it has no issues.
- [x] Resize - no issues found
- [ ] Interaction (https://github.com/sergiogval/sergiogval.github.io/pull/16/commits/1997cc4b4b99e2ca7f0168c809dc25e360038134)
- [x] Moving content - no issues found
- [x] Multimedia - no issues found
- [ ] The basic structure of the page (https://github.com/sergiogval/sergiogval.github.io/pull/16/commits/16d7adad5667e6c1685a07d7e64169e5bafb7661#diff-b78be019f1dc6d57753ea900c3805b114cd53ab7c0db836cc081836df1b99b7aL21)

A friendly reminder, the selected checkboxes were OK. In other words, they were already in compliance with accessibility best practices.

### CODE REVIEW

**Comment by sergiogval** [comment]
We reviewed all the projects of our team members and we changed the selected items of the list.
NOTE: The ones unselected were already in compliance with accessibility best practices.


- [x]     Page titles
- [x]     Image text alternatives** (alt text)
- [ ]     Text headings
- [ ]     Color contrast
- [ ]     Resize
- [x]     Interaction
- [ ]     Moving content
- [ ]     Multimedia
- [x]     The basic structure of the page

**Comment by rukundoeric** [comment]
Hi @sergiogval, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] Kindly move the report into PR description
  - [ ] In the result section of the report the you should include a statement pertained to every point checked:
          -  If everything was ok, they make a statement about that (e.g. "Page titles - no issues found").
          - If issues were found, **they should link to the corresponding line of code** in the Pull Request changes

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - N/A

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css [hunk 1]
```diff
@@ -11,14 +11,13 @@ body {
...
   background-color: transparent;
 
-  /* border: solid 2px red; */
   outline: none;
 }
 
 /* HEADER */
 .header {
   display: grid;
-  grid-template-columns: auto auto;
```
Comment (sergiogval) [inline style.css:21]: Change the units from auto to fr

File: index.html [hunk 2]
```diff
@@ -194,7 +194,7 @@ <h2 class="txt-black">About me</h2>
       <ul class="group-aboutme-cards">
         <li class="aboutme-cards">
           <img src="/images/languages.png" alt="Languages" class="imgabout" />
-          <h2>Languages</h2>
```
Comment (sergiogval) [inline index.html:211]: Change h2 elements to h3 when it applies.

File: index.html [hunk 3]
```diff
@@ -30,7 +30,7 @@
       href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css"
     />
   </head>
-  <body>
+  <body tabindex="0">
```
Comment (sergiogval) [inline index.html:33]: Add the interaction functionality

File: index.html [hunk 4]
```diff
@@ -152,7 +138,7 @@ <h3 class="txt-black">Multi-post Stories Gain+Glory</h3>
         <div class="portfolio-cards">
           <img
             src="./images/Untitled.png"
-            alt="portfolio-image"
```
Comment (sergiogval) [inline index.html:155]: Update alt text for image.

**Review by barackm — APPROVED** [review]
Hi @ahmedadel56, 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**